pulsar-broker/src/test/java/org/apache/pulsar/client/api/ClientDeduplicationTest.java

```diff
@@ -46,7 +64,7 @@ protected void cleanup() throws Exception {
         super.internalCleanup();
     }
 
-    @Test
+    @Test(priority = -1)
```

Comment:
Any reason for setting priority to `-1`?

Reply:
We need to run `testNamespaceDeduplicationApi` before all other tests because it assumes the initial duplication status is null.

```java
    public void testNamespaceDeduplicationApi() throws Exception {
        final String namespace = "my-property/my-ns";
        assertNull(admin.namespaces().getDeduplicationStatus(namespace));
```

After `testKeyBasedBatchingOrder` is added, `testNamespaceDeduplicationApi` will run after `testKeyBasedBatchingOrder`.